CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 [5.0.0-dev3]: https://github.com/microsoft/CCF/releases/tag/ccf-5.0.0-dev3
 
 - Updated Intel SGX PSW from 2.17 to 2.20 (#5616)
+- Secret sharing used for ledger recovery now relies on a much simpler implementation that requires no external dependencies. Note that while the code still accepts shares generated by the old code for now, it only generates shares with the new implementation. As a result, a DR attempt that would downgrade the code to a version that pre-dates this change, after having previously picked it up, would not succeed if a reshare had already taken place (#5655).
```

Comment:
Should we also add a note about this [here](https://microsoft.github.io/CCF/main/operations/recovery.html)?

Reply:
We can do but it's difficult to do it right without knowing in which version we will release this change. I think it's probably 4.0.9 and 5.0.0-dev2.